packages/neutrino-preset-jest/src/index.js

```diff
@@ -6,13 +6,25 @@ const { tmpdir } = require('os');
 const clone = require('lodash.clonedeep');
 const loaderMerge = require('neutrino-middleware-loader-merge');
 
+function getFinalPath(path) {
+  if (path[0] === '/') {
```

Comment:
It all depends on how you setup your aliases. If you use relative paths, then going off the <rootDir> will work fine, but if you use absolute paths, <rootDir>/absolutePath doesn't really work. Likewise, if you've aliased a module in node_modules, you'll need to explicitly same rootDir/node_modules/moduleName. Unless there's something off with my setup, loading aliases modules was probably never working...